specification/gedcom-03-datamodel.md

```diff
@@ -2347,6 +2353,11 @@ The `RESN` structure is provided to assist software in filtering data that shoul
 such that certain `RESN` structure payload entries result in the `RESN` structure and its superstructure being removed from the export.
 Such removal must abide by some constraints: see [Removing data](#removing-data) for more.
 
+A structure that points to a record that is being removed due to `RESN` processing will have its pointer replaced by a `voidPtr`,
+presering the structure's substructures.
+That might be desireable, but could cause accidental information leakage; for example, if a `g7:ADOP-FAMC` points to a restricted `FAMC` it may contain `FAMC`-identifying information in its substructures that the user also wishes to treat as restricted.
```

Comment:
```suggestion
preserving the structure's substructures.
That might be desirable, but could cause accidental information leakage; for example, if a `g7:ADOP-FAMC` points to a restricted `FAMC` it may contain `FAMC`-identifying information in its substructures that the user also wishes to treat as restricted.
```